src/platforms/dialogflow/facebook/FacebookEvent.ts

```diff
@@ -106,6 +93,21 @@ export class FacebookEvent extends DialogflowEvent {
       userId,
     };
   }
+
+  private getFacebookProfile(userFields: string) {
+    const params = [
```

Comment:
instead of adding querystring params like this, it's better to use `qs`

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/request/index.d.ts#L129
https://github.com/request/request#requestoptions-callback